src/services/dashboards/dashboard-detail/store/dashboard-detail-info.ts

```diff
@@ -203,6 +203,18 @@ export const useDashboardDetailInfoStore = defineStore('dashboard-detail-info',
         state.widgetDataMap = widgetDataMap;
     };
 
+    const updateWidgetInfo = (widgetKey: string, data: any) => {
+        const targetIndex = state.dashboardWidgetInfoList.findIndex((info) => info.widgetKey === widgetKey);
+        if (targetIndex > -1) {
+            const _dashboardWidgetInfoList = [...state.dashboardWidgetInfoList];
```

Comment:
nitpick: It's safer to use `cloneDeep`.

```suggestion
            const _dashboardWidgetInfoList = cloneDeep(state.dashboardWidgetInfoList);
```